Northstar.CustomServers/mod/scripts/vscripts/evac/_evac.gnut

```diff
@@ -421,7 +421,7 @@ void function EvacDropshipKilled( entity dropship, var damageInfo )
 {
 	foreach ( entity player in dropship.s.evacSlots )
 	{
-		if ( IsValid( player ) )
+		if ( IsValid( player ) && IsAlive( player ) )
```

Comment:
having dead players in evac slots at all is an issue, we should be preventing them from being in them ever rather than checking if there's somehow a dead player in an evac slot

Reply:
Dead players were killed again as I was making a custom modified version of Infection for someone else.

In that mod, I made it so Evac happens not during eGamestate.Epilogue but in eGamestate.Playing because otherwise win/lose doesn't count.

After the dropship made it to space, I set the winning team accordingly, however only those in the dropship were killed twice due to this line of code thus causing the crash.

It would be a lot easier to just add this one line of code rather than modifying gamestates, wouldn't it?

Reply:
> Dead players were killed again as I was making a custom modified version of Infection for someone else.
> 
> In that mod, I made it so Evac happens not during eGamestate.Epilogue but in eGamestate.Playing because otherwise win/lose doesn't count.
> 
> After the dropship made it to space, I set the winning team accordingly, however only those in the dropship were killed twice due to this line of code thus causing the crash.
> 
> It would be a lot easier to just add this one line of code rather than modifying gamestates, wouldn't it?

very late, but it doesn't really matter whether it's easier to add since having dead players in evac is an issue, even if this is an easy fix to your issue it still leaves bad behavior that's unwanted

Reply:
I could just revert this commit if you insist.